resources/views/atom.blade.php

```diff
@@ -23,8 +23,12 @@
             <summary type="html">
                 <![CDATA[{!! $item->summary !!}]]>
             </summary>
-            @if($item->__isset('enclosure'))
-              <enclosure url="{{ url($item->enclosure) }}" length="{{ $item->enclosureLength }}" type="{{ $item->enclosureType }}" />
+            @if(
+                isset($item->enclosure)
```

Comment:
Could we add a method `hasEnclosure` on `$item`, so we don't have to do a compound check here?